data/test/scenarios/test_return.cfg

```diff
@@ -15,3 +15,21 @@
         {RETURN ([not][true][/true][/not])}
     [/event]
 )}
+
+# Same as the above two, but using the Lua unit_test module
+{GENERIC_UNIT_TEST "test_lua_return" (
+    [event]
+        name = start
+        [lua]
+            code=<<unit_test.pass_if(true, 'return true')>>
+        [/lua]]
```

Comment:
There's an extra `]` on this line and on line 33, which means the preprocessor fails before any tests run.